src/scss/application.scss

```
@@ -126,6 +126,180 @@
+}
+
+// Column-mapping table: fixed layout so a very long user-supplied column name
+// (in a <select> option or a mapped cell) can't stretch the table past the page.
+.app-mapping-table {
+  table-layout: fixed;
+  width: 100%;
+}
+
+.app-mapping-table th,
+.app-mapping-table td {
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)